theme-base/_mixins.scss

```diff
@@ -1,3 +1,5 @@
+$focusOutlineOffset: 0 !default;
```

Comment:
The [comment here](https://github.com/primefaces/primevue-sass-theme/issues/45#issue-2069984281) mentions `$inputFocusOutlineOffset`.

Reply:
heh, good catch